src/script/chem/molfile/v3000.js

```diff
@@ -111,8 +111,25 @@ function parseBondLineV3000(line) {
 function v3000parseCollection(ctab, ctabLines, shift) {
 	/* reader */
 	shift++;
-	while (ctabLines[shift].trim() != 'M  V30 END COLLECTION')
+	while (ctabLines[shift].trim() != 'M  V30 END COLLECTION') {
+		const line = ctabLines[shift];
+		const atoms = line.substr(line.indexOf("ATOMS"), line.length);
+		const subsplit = splitonce(atoms, '=');
+		if (subsplit.length != 2)
+			throw new Error('A record of form AAA=BBB or AAA=(...) expected, got \'' + split[i] + '\'');
```

Comment:
nit: error message doesn't match the error condition. This error will throw for any string that doesn't contain at least one `=`.

Reply:
Yeah, I agree - the error checking condition is indeed written out loosely. But unfortunately they use this [everywhere else](https://github.com/schrodinger/ketcher/blob/master/src/script/chem/molfile/v3000.js#L141) too.

Maybe we can fine-grain the error checking condition as part of the clean-up ticket https://jira.schrodinger.com/browse/SS-29875.
Have added this point in the ticket description too.